rollup.config.js

```diff
@@ -1,22 +1,49 @@
-import babel from 'rollup-plugin-babel';
+import babel from "rollup-plugin-babel";
+import filesize from 'rollup-plugin-filesize';
+
+import pkg from "./package.json";
+const { keys } = Object;
+
+const globals = {
```

Comment:
Having to explicitly enumerate all dependencies here in a rollup config seems fragile to me. Is this normally what you have to do?

Reply:
I don't know that much about "normal" use, but from what I understand, UMD is compatible with globals where ramda package is available on R namespace. In global mode, there is no way to declare where modules are, so you have to explicitly tell Rollup how to match imports to globally available functions.

Reply:
Btw, tests will now complain if something is missing from Globals because rollup happens when tests run.